tests/unit/test.program.js

```diff
@@ -323,4 +324,13 @@ describe('program.defaultVersionGetter', () => {
       });
   });
 
+  it('returns the development package version', () => {
+    process.env.NODE_ENV = 'development';
+    let root = path.join(__dirname, '..', '..');
+    let git = require('git-rev-sync');
+    var devVersion = `${git.branch()}-${git.long()}`;
```

Comment:
Always declare variables with `const` (constant) when you don't need to modify them. This will help catch any unintended modifications.